services/director-v2/tests/unit/with_dbs/test_utils_dask.py

```diff
@@ -75,14 +74,19 @@ def service_version() -> str:
     return "1234.32432.2344"
 
 
-@pytest.fixture()
-def project_id() -> ProjectID:
-    return uuid.uuid4()
+@pytest.fixture
+def user_id(faker: Faker) -> UserID:
+    return faker.pyint(min_value=1)
 
 
-@pytest.fixture()
-def node_id() -> NodeID:
-    return uuid.uuid4()
+@pytest.fixture
+def project_id(faker: Faker) -> ProjectID:
+    return ProjectID(faker.uuid4())
```

Comment:
you can return an UUID from the method, there should be a parameter for it

Reply:
I actually hate the documentation of faker... I can never find anything